examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+## Arguments:
+# $1: number of timesteps to run
+# $2: number of ranks to execute with (ensure that this is compatible with fv3core)
+# $3: backend to use in gt4py
+# $4: target directory to store the output in
+# $5: path to the data directory that should be run
+#############################################
+# Example syntax:
+# ./run_on_daint.sh 60 6 gtx86
+
+exitError()
+{
+    echo "ERROR $1: $3" 1>&2
+    echo "ERROR     LOCATION=$0" 1>&2
+    echo "ERROR     LINE=$2" 1>&2
+    exit $1
+}
+
+SCRIPT=`realpath $0`
```

Comment:
This is for daint only (run_on_daint), so this is fine, just want to note this isn't present on Macs. But what would the point be of benchmarking on our macs? hahahha

Reply:
good to know